examples/lib/getSlot.ts

```diff
@@ -0,0 +1,30 @@
+import { ChainId } from '@/utils';
+import { TOKENS } from 'test/lib/utils';
+import { Address } from 'viem';
+
+export function getSlot(chainId: ChainId, tokenAddress: Address): number {
```

Comment:
Nice helper 👏 
It would be nice to have a small description of what slot means, since it's such a low level thing, right?
Something like:
> slot refers to the storage slot responsible for token balance within an ERC20 contract. It is used to artificially manipulate account balance before any tests/examples on local forks.